### ISSUE TITLE
request: add a getting started section to doc.

### ISSUE THREAD

**Issue by tiarno** [holloway/doctored#14]
Just a request to show a quick getting started + installation instructions in the doc.
thanks,

**Comment by holloway** [comment]
Good idea. I'll write it in a few weeks, or accept any patches for that. Cheers
